src/pipeline_edsl/to_workflow.ml

```diff
@@ -639,12 +642,20 @@ module Make (Config : Compiler_configuration)
         ~output:(`CSV output_file)
     )
 
-  let vaxrank ?(configuration=Tools.Vaxrank.Configuration.default)
+  let vaxrank
+      ?(configuration=Tools.Vaxrank.Configuration.default)
       reference_build vcf bam predictor alleles =
     let v = get_vcf vcf in
     let b = get_bam bam in
     let mhc = get_mhc_alleles alleles in
-    let outdir = Config.work_dir in
+    let outdir =
```

Comment:
this should be a directory instead of a file. Vaxrank will put a CSV and TXT file in that folder with default names: `vaxrank-result.csv` and `vaxrank-result.txt`.